portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,25 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private List<String> fruits;
```

Comment:
Since it won't ever change let's use ImmutableList
https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableList.html

 private static final ImmutableList<String> fruits = ImmutableList.of("Mango", "Strawberry", "Banana");

Reply:
Sounds good, thank you!